### PR TITLE
Cleanup II

### PR DESCRIPTION
**Pull request by the-maldridge**
This set mainly just cleans up the netboot script.  I wanted to split this off from the 3rd set which will clean up the mklive script.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,38 +47,49 @@ dist: distdir-$(DATE)
 rootfs-all: $(ALL_ROOTFS)
 
 rootfs-all-print:
-	echo $(ALL_ROOTFS)
+	@echo $(ALL_ROOTFS) | sed "s: :\n:g"
```
Comment (Gottox) [inline Makefile]: `tr ' ' '\n'`

File: mkrootfs.sh.in [hunk 2]
```diff
@@ -206,7 +208,7 @@ rm -rf "$ROOTFS/var/cache/*" 2>/dev/null
 
 # Finally we can compress the tarball, the name will include the
 # architecture and the date on which the tarball was built.
-tarball=void-${XBPS_TARGET_ARCH}-ROOTFS-$(date '+%Y%m%d').tar.xz
+tarball=${FILENAME-void-${XBPS_TARGET_ARCH}-ROOTFS-$(date '+%Y%m%d').tar.xz}
```
Comment (Gottox) [inline mkrootfs.sh.in]: shouldn't it be `${FILENAME:-...` instead of `${FILENAME-...`?

Comment (Gottox) [reply mkrootfs.sh.in]: Idea: replace the tarball variable completely and replace it by
```bash
: ${FILENAME:=void-${XBPS_TARGET_ARCH}-ROOTFS-$(date '+%Y%m%d').tar.xz}
```

File: Makefile (hunk 1 above)
Comment (the-maldridge) [reply Makefile]: I can see where you're coming from with this, but nothing else in the entire suite pulls in `tr`.  I'm trying to keep this low dependency if at all possible, but if this is something you really think should be changed, I can go ahead and swap it out.

File: mkrootfs.sh.in (hunk 2 above)
Comment (the-maldridge) [reply mkrootfs.sh.in]: the single dash form works as well, but I like the solution you proposed of just removing a variable.

File: Makefile (hunk 1 above)
Comment (Gottox) [reply Makefile]: Ok 👍

**Pull request merged**